test/calculation/GravityModelVectorUtility.cpp

```diff
@@ -3,6 +3,7 @@
 namespace polyhedralGravity {
 
     std::vector<Array3Triplet> GravityModel::calculateSegmentVectors(const Polyhedron &polyhedron) {
+        using namespace detail;
```

Comment:
not the expert here but there must be a smarter way then spending 13 lines of this file with reiterating the exact same using? :D

Reply:
Indeed 😇